### ISSUE TITLE
break on boolean comparator (coffescript)

### ISSUE THREAD

**Issue by gagalago** [adrianlee44/atom-aligner#39]
this example
```coffee
aaaaaaaaaaaaaaaaa = "a"
while b != "b"
```
is miss aligned to 
```coffee
aaaaaaaaaaaaaaaaa = "a"
while b !         = "b"
```

**Comment by adrianlee44** [comment]
Oh no! This is a regression. Looking into this.

**Comment by adrianlee44** [comment]
It seems like there is a bug with atom/language-coffee-script where `!` has a higher priority than `!=`.

**Comment by adrianlee44** [comment]
This is fixed with Atom 1.2 release which is using the latest language-coffee.
